js/app/demo/app.jsx

```diff
@@ -157,8 +150,41 @@ define(['react', 'jsx!editor', 'jsx!messages', 'jsx!fixedCode', 'jsx!configurati
                         </div>
                         <div className="col-md-5">
                             <ul className="nav nav-tabs" role="tablist">
-                                <li role="presentation" className="active"><a href="#messages" aria-controls="messages" role="tab" data-toggle="tab">Messages</a></li>
-                                <li role="presentation"><a href="#fixedCode" aria-controls="fixedCode" role="tab" data-toggle="tab">Fixed Code</a></li>
+                                <li
+                                    role="presentation"
+                                    className={this.state.fix ? "" : "active"}
+                                >
+                                    <a
+                                        href="#messages"
+                                        aria-controls="messages"
+                                        role="tab"
+                                        onClick={
+                                            function() {
+                                                this.setState({ fix: false });
+                                            }.bind(this)
```

Comment:
The value of these `onClick` props will be reference-unequal every time this component renders since it's binding a new function very time. What would you think of converting these to handlers on the class? The real performance impact will be negligible, but I wanted to point it out anyway.